### PR TITLE
La recette of le pain de campagne

### PR DESCRIPTION
**Pull request by basilesimon**

### CODE REVIEW

File: pain-campagne.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title: pain de campagne
+---
+
+# Le Pain de campagne
+
+![](img/le_pain/result.JPG)
+
+## Day one
+
+Prepare a _stiff starter:_
```
Comment (Cgg) [inline pain-campagne.md:12]: `_stiff starter_:` non ? Quelle la règle typographique dans ce cas de figure?

**Review by Cgg — COMMENTED** [review]
I will try this and merge once successful.

File: pain-campagne.md (hunk 1 above)
Comment (basilesimon) [reply pain-campagne.md:12]: [as per this](https://graphicdesign.stackexchange.com/questions/48540/in-a-sentence-ending-with-a-bold-or-italicised-word-does-the-period-need-to-be)

> all punctuation marks should be printed in the same style or font of type as the word or letter immediately preceding them.

**Review by Martiusweb — APPROVED** [review]
top!

**Pull request merged**